i18n/ar.json

```diff
@@ -1378,4 +1376,4 @@
   "yes": "نعم",
   "you_dont_have_any_shared_links": "ليس لديك أي روابط مشتركة",
   "zoom_image": "تكبير الصورة"
-}
+}
```

Comment:
You removed all the trailing new lines in the i18n files. Please fix that as well (or as Alex said just revert the changes entirely, you don't need to fix any language besides English).

Reply:
VScode had the great idea to automatically remove newlines without asking. I ended up reverting all the json files, so it should be fine now. Not sure why the weblate test is crashing, so I hope the newline stuff fixes it.